### PR TITLE
Issue #70 move the minishift binary to $(BIN_DIR) after extracting

### LINKED ISSUE #70: Update downloading of minishift binary to handle tar.gz subdirectory change
With commit https://github.com/minishift/minishift/commit/868b6759329ceb1baa0613e29835b5d69d6aaee7 , all content will be under a subdirectory.

Update [Makefie](https://github.com/minishift/minishift-b2d-iso/blob/master/Makefile#L40-L46) post 1.7.0 release of minishift.

### PR DESCRIPTION
**Pull request by anjannath**
Fix #70

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,6 +9,7 @@ OSRELEASE_FILE=os-release
 OSRELEASE_TEMPLATE=os-release.template
 MINISHIFT_LATEST_URL=$(shell python tests/utils/minishift_latest_version.py)
 ARCHIVE_FILE=$(shell echo $(MINISHIFT_LATEST_URL) | rev | cut -d/ -f1 | rev)
+MINISHIFT_DIR=$(shell echo $(MINISHIFT_LATEST_URL) | rev | cut -d/ -f1 | cut -c5- | rev)
```
Comment (brgnepal) [inline Makefile]: @anjannath , plz update `11-12` lines with 
```
ARCHIVE_FILE=$(shell echo $(MINISHIFT_LATEST_URL) | cut -d/ -f9-11)
MINISHIFT_UNTAR_DIR=$(shell echo $(ARCHIVE_FILE) | sed 's/.tgz//')
```

File: Makefile [hunk 2]
```diff
@@ -43,6 +44,9 @@ $(BIN_DIR)/minishift:
 	@cd $(BIN_DIR) && \
 	curl -LO --progress-bar $(MINISHIFT_LATEST_URL) && \
 	tar xzf $(ARCHIVE_FILE)
+	@echo "Moving minishift binary to $(BIN_DIR)"
```
Comment (brgnepal) [inline Makefile]: Update line `42` with `@echo "Downloading latest minishift binary at $(BIN_DIR)/minishift"`. This one is not required.

**Review by brgnepal — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (brgnepal) [reply Makefile]: @anjannath , CI is also failing. The another cd to $(BIN_DIR) is not working fine. Check

File: Makefile [hunk 3]
```diff
@@ -38,11 +39,12 @@ release: iso $(GOPATH)/bin/gh-release
...
 
 $(BIN_DIR)/minishift:
-	@echo "Downloading latest minishift binary..."
+	@echo "Downloading latest minishift binary at $(BIN_DIR)/minishift..."
 	@mkdir -p $(BIN_DIR)
 	@cd $(BIN_DIR) && \
 	curl -LO --progress-bar $(MINISHIFT_LATEST_URL) && \
-	tar xzf $(ARCHIVE_FILE)
+	tar xzf $(ARCHIVE_FILE) && \
+	mv $(MINISHIFT_UNTAR_DIR)/minishift .
```
Comment (brgnepal) [inline Makefile:47]: :+1:  This should work.

**Comment by brgnepal** [comment]
@anjannath , Please update the commit message with capitalized issue `Issue #...`.

**Comment by anjannath** [comment]
Sorry @budhrg i havn't been watching this repo, so missed the comment, updating now.

**Comment by brgnepal** [comment]
retest this please

**Comment by brgnepal** [comment]
@anjannath , need rebase.

**Comment by anjannath** [comment]
@budhrg I have no idea why its failing. Please have a look when you get time.

**Comment by brgnepal** [comment]
retest this please

**Review by brgnepal — APPROVED** [review]

**Comment by brgnepal** [comment]
Merged and thanks.

**Pull request closed without merging**